erpnext/controllers/accounts_controller.py

```diff
@@ -1235,6 +1236,12 @@ def validate_and_delete_children(parent, data):
 
 		d.cancel()
 		d.delete()
+		
+		from erpnext.stock.stock_balance import update_bin_qty, get_ordered_qty
+		frappe.errprint(f"Item Code: {d.item_code}, Warehouse: {d.warehouse}")
+		update_bin_qty(d.item_code, d.warehouse, {
+			"ordered_qty": get_ordered_qty(d.item_code, d.warehouse)
+		})
```

Comment:
- will have to include `reserved_qty` too as the same issue goes for Sales Order.
- Remove the `frappe.errprint`

Reply:
Agree, have added the `reserved_qty`, removed "frappe.errprint".